backend/controllers.py

```diff
@@ -0,0 +1,164 @@
+import bottle
+import json
+import rauth
+import config
+import models
+import logging
+from models import User
+
+user_info_uri = ('https://www.googleapis.com/oauth2/v1/userinfo')
+oauth2 = rauth.OAuth2Service
+google = oauth2(
+    client_id=config.GOOGLE.CLIENT_ID,
+    client_secret=config.GOOGLE.CLIENT_SECRET,
+    name='google',
+    authorize_url='https://accounts.google.com/o/oauth2/auth',
+    access_token_url='https://accounts.google.com/o/oauth2/token',
+    base_url='https://accounts.google.com/o/oauth2/auth',
+)
+
+if config.DEVELOPMENT:
+    redirect_uri = '{uri}:{port}/api/success'.format(
+                   uri="http://" + config.HOST,
+                   port=config.PORT)
+else:
+    redirect_uri = config.BASE_URI + "/api/success"
+
+page = open('../index.html', "r").read()
+
+
+class StaticFiles:
+    def __init__(self):
+        @bottle.route('/site/<path:re:(images|css|js)\/.+>')
+        def server_static(path):
+            return bottle.static_file(path, root='site')
+
+
+class Routes:
+    def __init__(self):
+        @bottle.route('/', methods=['GET'])
+        def index():
+            return page
```

Comment:
Yeah it could, however it would be accessed with http://automata.discrete.gr/index.html
Maybe I could do that and change the apache conf for server root?